core/arch/arm/plat-aspeed/conf.mk

```diff
@@ -0,0 +1,25 @@
+PLATFORM_FLAVOR ?= ast2600
+
+ifeq ($(PLATFORM_FLAVOR),ast2600)
+
+include core/arch/arm/cpu/cortex-a7.mk
+
+$(call force,CFG_8250_UART,y)
+$(call force,CFG_ARM32_core,y)
+$(call force,CFG_TEE_CORE_NB_CORE,2)
+$(call force,CFG_TEE_CORE_DEBUG, y)
+$(call force,CFG_TEE_CORE_LOG_LEVEL, 4)
+$(call force,CFG_GIC,y)
+$(call force,CFG_SECURE_TIME_SOURCE_CNTPCT, y)
+
+CFG_NUM_THREADS ?= CFG_TEE_CORE_NB_CORE
+
+CFG_DRAM_BASE ?= 0x80000000
+CFG_DRAM_SIZE ?= 0x40000000
+
+CFG_TZDRAM_START ?= 0x88000000
+CFG_TZDRAM_SIZE ?= 0x01000000
+
+CFG_SHMEM_START ?= ($(CFG_TZDRAM_START) + $(CFG_TZDRAM_SIZE))
```

Comment:
You don't have to have this in case you're using `CFG_CORE_DYN_SHM=y`.
With this you have to carve out this range from the memory available to the kernel. So this has to be kept in sync with the DT passed to the kernel. This can be disabled with `CFG_CORE_RESERVED_SHM=n`.

On the other hand if you want to use `CFG_CORE_RESERVED_SHM=y` it will only be activated in the communication with the kernel if `CFG_CORE_DYN_SHM=n` or in case the kernel fails to initialize dynamic shared memory.

Reply:
Thanks for the suggestion.
I will try the dynamic share memory scenario for better memory flexbility.
The revision will be verified and submitted soon.

Reply:
Revise as suggested with 4c79cba9a3f7d0e25923a222312627047b46f645.
It is also verified with a Linux kernel without carving a share memory region.